pkg/server/v3/server.go

```diff
@@ -220,6 +222,10 @@ func (s *server) StreamExtensionConfigs(stream extensionconfigservice.ExtensionC
 	return s.StreamHandler(stream, resource.ExtensionConfigType)
 }
 
+func (s *server) StreamRlsConfigs(stream rlsconfigservice.RateLimitConfigDiscoveryService_StreamRlsConfigsServer) error {
```

Comment:
Can we add a method for the Delta RLS stuff as well? Or does RLS not support delta yet?

Reply:
Rls does not support delta yet.

Reply:
Cool can you make an issue to track that?

Reply:
created an issue: https://github.com/envoyproxy/ratelimit/issues/393